sdk-api-src/content/cfapi/nf-cfapi-cfcreateplaceholders.md

```diff
@@ -56,9 +56,7 @@ Creates one or more new placeholder files or directories under a sync root tree.
 
 ### -param BaseDirectoryPath [in]
 
-Local directory path under which placeholders are created.
-
-Note that this directory must be the immediate parent directory of the placeholders being created. For example, if the sync root of the provider is c:\SyncRoot and a placeholder is being created in a child directory of the sync root such as c:\SyncRoot\ChildDir, then BaseDirectoryPath = c:\SyncRoot\ChildDir.
+Path to the local directory in which the placeholders are created. This path must be under the sync root of the provider.
```

Comment:
"This path must be under the sync root of the provider."

Is this the same problem with using the word under since it might in the sync root?

Reply:
The earlier description was "Local directory path under which placeholders are created." - Now under here referred to the placeholders as that could mean this directory or any descendent of this directory. 

"This path must be under the sync root of the provider." - This refers to the the fact that the directory (where the placeholders are to be created) can be any descendent of the SyncRoot. 

So in my opinion, this may be fine. I am open to suggestions though. @oldnewthing what's your take here ?

Reply:
Yeah, this is okay. "under" means "any descendant". The base directory can be any descendant of the sync root (so "under" is correct in the second sentence). But it must be the immediate parent of the placeholder (so "under" is incorrect in the first sentence.)